src/pages/accessibility.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement } from 'react';
+import { TwoThirdsLayout } from '../layout/Layout';
+
+const title = 'Accessibility - Fares Data Build Tool';
+const description = 'Accessibility page for the Fares Data Build Tool';
+
+const Accessibility = (): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description}>
+            <h1 className="govuk-heading-l">Accessibility statement for Fares Data Build Tool</h1>
```

Comment:
Should we have a 'the' in here? Reads a bit weird without I think - just my opinion though.